tests/test-sources/plugins/lsp/omnisharp.nix

```diff
@@ -1,13 +1,10 @@
-{ pkgs, ... }:
-{
+_: {
```

Comment:
Unfortunate formatting... Maybe just leave the `pkgs` argument for now, even though its unused?

Reply:
Or if there's no plan to add it back (was it just being used to access `stdenv`?), we could drop the functional-style entirely for a plain attrset.

Reply:
```suggestion
{
```